### PR TITLE
Added pagination example with buttons

### PR DESCRIPTION
**Pull request by Prince2347X**
## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [X] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: examples/views/pagination.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import discord
+from discord.ext import commands
+
+
+class PaginationView(discord.ui.View):
+    current = 0 # It's not in the constructor because it'll be reset everytime the view is updated.
```
Comment (Phxntxm) [inline examples/views/pagination.py:6]: No it wouldn't? If it would, so would the `embed_list`

File: examples/views/pagination.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    # To make sure only ctx.author can interact.   
+    async def interaction_check(self, interaction: discord.Interaction) -> bool:
+        if self.user == interaction.user:
+            return True
+        await interaction.response.send_message(f'Only `{self.user.name}` can interact. Run the command if you want to.', ephemeral=True)
+        return False
+        
+    @discord.ui.button(label="Previous", style=discord.ButtonStyle.red, disabled=True) # you can add emojis too..
+    async def previous(self, button, interaction):
```
Comment (Phxntxm) [inline examples/views/pagination.py:24]: Why are some things type hinted, but some aren't

File: examples/views/pagination.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        self.current -= 1
+        self.next.disabled = False
```
Comment (Phxntxm) [inline examples/views/pagination.py:26]: Why is this here? Is this implying that `embed_list` can be modified and next should be updated to reflect that? If so, then there is no handling of to disabling the button later

File: examples/views/pagination.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        self.current += 1
+        self.previous.disabled = False
+        self.previous.style = discord.ButtonStyle.green
+        self.count.label = f"Page {self.current + 1}/{len(self.embed_list)}"
+        if self.current == len(self.embed_list) -1 :
+            button.disabled = True
+            button.style = discord.ButtonStyle.red
+            await interaction.response.edit_message(embed=self.embed_list[self.current], view=self)
+        else:
+            await interaction.response.edit_message(embed=self.embed_list[self.current], view=self)
```
Comment (Phxntxm) [inline examples/views/pagination.py:53]: In general, this is nearly the same exact thing as `previous`. This should really be moved into a function that handles updating information

**Review by Phxntxm — CHANGES_REQUESTED** [review]

**Comment by Prince2347X** [comment]
Noted.

**Comment by Rapptz** [comment]
Sorry, this example doesn't match the style and quality of the pre-existing examples.

**Pull request closed without merging**